### ISSUE TITLE
4.12.0 Regression: Using cypress does not display success message when using open

### ISSUE THREAD

**Issue by karinecord** [cypress-io/cypress#9031]
Hi,
While I was writing a test to verify if a message is displayed, I've noticed that cypress is behaving strangely. 
When the app form is submitted, it should display a success messsage, but on v4.12.0 does not display.
Initially, I've thought it would be the app I was testing, but when I submit the form manually it displays as expected.
Started with v5.3.0, than I've figured that until version 4.11.0 it works fine, but from 4.12.0 the issue begins.

**Cypress 4.11.0**
![test-message-v4 11 0](https://user-images.githubusercontent.com/15333795/97630814-fc11be00-1a27-11eb-98d4-d638102b5dd6.png)

**Cypress 4.12.0**
![test-message-4 12 0](https://user-images.githubusercontent.com/15333795/97630825-016f0880-1a28-11eb-9aa6-7a1e38d89c36.png)

It's strange because it fails only when using `cypress open`. 
Using `cypress run` or `cypress run --browser chrome` works fine. Plus, the console displays loads of `GET` requests.

Here it's the code that reproduces the issue https://github.com/karinecord/cypress-regression-test

```
npm install
./node_modules/.bin/cypress open
```

Doing the code below the test pass:

```
npm uninstall cypress --save && npm install cypress@4.11.0 --save
./node_modules/.bin/cypress open
```

## Tecnical details
Browser:
Chrome Version 86.0.4240.75 (Official Build) (64-bit)

OS
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.5 LTS
Release:	18.04
Codename:	bionic

Node: v10.15.1

**Comment by jennifer-shehane** [comment]
I can recreate this issue from the repo provided. This was introduced in 4.12.0 by this PR: https://github.com/cypress-io/cypress/pull/8095

### Repro

```js
it('should display sucess message', () => {
  cy.visit('https://opensource-demo.orangehrmlive.com/');

  cy.get('#txtUsername').type('Admin');
  cy.get('#txtPassword').type('admin123');
  cy.get('#btnLogin').click();

  cy.get('#menu_pim_viewMyDetails').click();
  cy.get('#sidenav').contains('Contact Details').click();

  cy.get('#btnSave').click();
  cy.get('#btnSave').click();

  cy.get('.message.success').contains('Successfully Saved').should('be.visible');
});
```

**Comment by karinecord** [comment]
Great, thanks @jennifer-shehane !

**Comment by bahmutov** [comment]
Released in v6.1.0
